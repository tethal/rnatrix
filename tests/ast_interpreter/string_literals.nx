fun main() {
    print("hello");
    print("world");
    print("");
    print("quote: \"");
    print("backslash: \\");
    print("newline: \n");
    return 0;
}