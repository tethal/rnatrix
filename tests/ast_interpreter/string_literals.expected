hello
world

quote: "
backslash: \
newline:

